doc/manual.md

```diff
@@ -55,29 +53,27 @@ return values for functions.
 
 ### Records
 
-Records are nominal and should be declared in the top level, like the following
-example.
+Records types in Titan are nominal and should be declared in the top level,
+following the syntax `record Type (field: type)* end`. The following example
+declare a record `Point` with the fields `x` and `y` which are floats.
 
     record Point
         x: float
         y: float
     end
 
-    record Circle
-        center: Point
-        radius: float
-    end
-
-After the top level declaration, you may create records with the `.new`
-constructor:
-
-    local p = Point.new(1, 2)
-    local c = Circle.new(p, 3.5)
-
-You can access the fields with the dot operator:
-
-    local a = p.x
-    p.y = 2
+You can create records with initializer lists, assigning a value to each field
+either by it's name or positional order.
+For instance, you could initalize an instance of the record `Point` with: `{ x =
+3, y = 5 }`, `{ y = 5, x = 3 }` or `{ 3, 5 }`.
```

Comment:
-1 to `{ 3, 5 }` — very misleading!

Reply:
I think it is important that it be possible to instantiate records without needing to type all the field names. I don't have a strong preference between a Lua-like `local p = Point.new(3,5)` or a C-like `local p : Point = {3, 5}` though. Just don't make me write `local p : Point = Point.new(3,5)`  :)

Reply:
Removed the `{ 3, 5 }` syntax for records. Now we have different ast nodes for arrays constructors and records initalizer lists, so things like `{ y = 5, 3 }` are syntax errors.